.pre-commit-config.yaml

```
@@ -23,8 +23,11 @@ repos:
       - id: mixed-line-ending
       - id: name-tests-test
         args: ["--pytest-test-first"]
+      - id: no-commit-to-branch #default is master and main
+        name: branch is not named `main`. Fix is `git branch -m <newname>`
```

I think the current version makes it sound like my branch _should_ be named main if it fails.

```suggestion
      - id: no-commit-to-branch # default is master and main
        name: branch should not be named `main`.
```
or "use a feature branch".

If you're adding the better/more complete doc link to the PR cleanliness check then perhaps we should just leave the fix portion there instead of cluttering the pre-commit with a potential solution? I'm in favor of short and concise, not everything needs to be complete in these quick checks IMO.